backend/infrahub/core/attribute.py

```diff
@@ -599,6 +600,25 @@ class Boolean(BaseAttribute):
     type = bool
 
 
+class DateTime(BaseAttribute):
+    type = str
+
+    @classmethod
+    def validate_format(cls, value: Any, name: str, schema: AttributeSchema) -> None:
+        super().validate_format(value=value, name=name, schema=schema)
+
+        if not value and schema.optional:
+            return
+
+        try:
+            time = pendulum.parse(value, exact=True)
```

Comment:
Sorry a bit late to the party here, but I'd suggest that we use the "internal" Timestamp instead of using pendulum directly. Mostly to be consistent with how this is done elsewhere. I.e. from infrahub.core.timestamp (or from infrahub_sdk import Timestamp)

Reply:
That makes a lot of sense and has been implemented.